config/jobs/etcd/etcd-presubmits.yaml

```diff
@@ -236,3 +236,44 @@ presubmits:
           limits:
             cpu: "4"
             memory: "8Gi"
+
+  - name: pull-etcd-robustness-amd64
+    cluster: eks-prow-build-cluster
+    optional: true # remove this once the job is green
+    always_run: true
+    branches:
+      - main
+    decorate: true
+    decoration_config:
+      timeout: 60m
+    annotations:
+      testgrid-dashboards: sig-etcd-presubmits
```

Comment:
I think we can also add it to the robustness dashboard

Reply:
Done 👍

Reply:
Sorry if I wasn't clear, but I meant to add rather than replace. It can appear in more than one dashboard.

Reply:
Yeah, I misread that, adding it to both.